python/lsst/ts/standardscripts/maintel/m1m3/check_hardpoint.py

```diff
@@ -0,0 +1,89 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["CheckHardpoint"]
+
+from lsst.ts.idl.enums.MTM1M3 import HardpointTest
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+
+from lsst.ts import salobj
+
+
+class CheckHardpoint(salobj.BaseScript):
+    """Check M1M3 Individual hardpoint breakaway.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    - "Testing hardpoint {id}.": Check hardpoint.
+    - "Hardpoint breakaway check complete.": Check complete.
+
+    """
+
+    def __init__(self, index, add_remotes: bool = True):
+        super().__init__(index=index, descr="Check M1M3 Hardpoint")
+
+        self.mtcs = MTCS(
+            domain=self.domain,
+            intended_usage=None if add_remotes else MTCSUsages.DryTest,
+            log=self.log,
+        )
+
+    @classmethod
+    def get_schema(cls):
+        return None
+
+    async def configure(self, config):
+        # This script does not require any configuration.
+        pass
+
+    def set_metadata(self, metadata):
+        metadata.duration = 180.0
+
+    async def wait_hardpoint_test_ok(self):
+        """Wait for the laser to warm up and be ready to be used."""
+
+        self.log.info("Checking if the bump test has passed.")
```

Comment:
It's not a bump test. It's a hardpoint breakaway test. Please change the message to:

"Checking if the hardpoint breakaway test has passed."